hell/adventcalendar/2025/34/index.md

```diff
@@ -1,35 +1,104 @@
+</figure>
+
+<figure>
+
+![Screenshot of CKEditor5](https://www.drupal.org/files/issues/2021-09-04/cke5_dialogless_link_editing.png)
+
+<figcaption>CKEditor5, which is used in the Drupal Core, has several formatting options in the editing interface. Among many options, the 'bold' and 'italic' options are visible, which will output 'strong' and 'em' HTML tags. There are no separate options for 'b' or 'i' tags.</figcaption>
+</figure>
+
+So even when a content editor might know about the difference, there is no way to use `<b>` and `<i>` tags.
+
+## What is the difference?
+
+The key difference between these tags lies in how a person perceives the text: <b>visually or non-visually</b>. The problem is that right now, in many cases, `<strong>` and `<em>` tags are used as one single solution to two different approaches. You should only use `<strong>` and `<em>` tags if you really want to signal importance or emphasise a text, both when visually reading it and when the text is read to you. By assistive technology, for example.
+
+Semantics and accessibility overlap but are not identical: semantics describe meaning, while accessibility ensures that meaning is conveyed appropriately to all users
+
+If you feel the need to use an HTML element for extra emphasis, ask yourself this:
+
+"Would a person also want to hear this emphasis loud and clear, by stressing specifically these words when the page is read aloud to them?"
+
+If the answer is no: congratulations! You found a use case for `<b>` and `<i>` tags. Or you could use any other HTML element and apply custom `font-weight` styling. This is exactly what the WCAG auditor meant with the feedback we got. You intend to make the text only _visually_ emphasized, not audibly.
+
+If the answer is still yes: hang on for a second and read on to learn why you still might want to choose `<b>` or `<i>`.
+
+## Why strong/em can lead to worse accessibility
+
+The issue with the current usage of `<strong>` and `<em>` tags is that they are too often heavily over-used. Consider an example text in HTML like this:
 
 ```html
-<h1> 
-  <a href="/">
-    Hello World
-  </a>
-</h1>
+We hereby would like to invite <strong>all</strong> of our colleagues and
+clients to a <strong>special event</strong> that will only take place
+<strong>once every two years</strong>. During this
+<strong>full-day event</strong> we will visit the
+<strong>beautiful city center</strong> and have a spectacular dinner at the
+restaurant serving <strong>100% local food</strong>.
 ```
+
+Of course I'm exaggerating a bit here, but these kinds of texts happen. This is why our websites get flagged by auditors. Imagine having to hear this text read aloud; would you really want to hear all these words with alternating stress in one sentence? This is exactly the problem. There is too much emphasis on too many words. The non-visual user is almost harassed by the amount of importance it hears.
```

Comment:
This kinda adds to my previous comment: Isn't this also an issue for visual users? If everything is important, nothing is. The real problem isn't how it's potentially announced by AT, but that they're using the element incorrectly. Again, telling editors to use a different element that looks the same isn't a solution. In most cases they would be confused or unsure.